lib/auth/legacy.js

```diff
@@ -57,6 +58,7 @@ const login = async (opts) => {
   }
 
   try {
+    output(`Log in on ${opts.registry}`)
```

Comment:
I believe it might be better to use `npmlog.notice` here instead 🤔

Reply:
Yeah I have no complaints with changing this out I just assumed we wouldn't want the extra text the notice adds before it, I'll change this up ASAP!